packages/nextjs/components/HooksExample.tsx

```diff
@@ -0,0 +1,70 @@
+import { ReactNode } from "react";
+
+const Red = ({ children }: { children: ReactNode }) => {
+  return <span className="text-[#FF5962]">{children}</span>;
+};
+const Cyan = ({ children }: { children: ReactNode }) => {
+  return <span className="text-[#00B0A6]">{children}</span>;
+};
+
+const Purple = ({ children }: { children: ReactNode }) => {
+  return <span className="text-[#7B61FF]">{children}</span>;
+};
+export const HooksExample = () => {
+  return (
+    <div className="w-full flex-grow  bg-white rounded-2xl p-6 drop-shadow-[0px_0px_10px_rgba(139,178,241,0.50)]">
+      <div className="bg-base-300/40 w-full h-full border border-primary rounded-3xl p-4 lg:p-6 text-xs lg:text-sm  font-mono overflow-x-scroll whitespace-nowrap  lg:overflow-auto lg:whitespace-normal">
+        <p className="my-3">
+          <Red>import</Red>
+          {" { "} useScaffoldContractWrite {" } "}
+          <Red>from</Red> <Cyan>&quot; ~~/hooks/scaffold-eth &quot;</Cyan>;
```

Comment:
I think there is extra space before and after the path:

```suggestion
          <Red>from</Red> <Cyan>&quot;~~/hooks/scaffold-eth&quot;</Cyan>;
```

Reply:
Yup 🙌